model/host/host.go

```diff
@@ -435,35 +435,41 @@ func IsIntentHostId(id string) bool {
 	return strings.HasPrefix(id, "evg-")
 }
 
-func (h *Host) SetStatus(status, user string, logs string) error {
-	if h.Status == status {
+// SetStatus updates a host's status on behalf of the given user.
+func (h *Host) SetStatus(newStatus, user, logs string) error {
```

Comment:
Is this used anywhere else? If not, wouldn't it be simpler to have a single `SetFields` func that is called by `Terminate` with all of the fields we want to update?

Reply:
Yes, `SetStatus` is used for most of the other status updates, which don't involve setting additional fields.